app/controllers/YourIncomeCalculationControllerNew.scala

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Can you rename this to YourIncomeCalculationController? Or do you have a toggle to switch between the two? If so i am guessing you will be renaming this after you are done with the toggle?

Reply:
Yes, Ramin, Currently we are giving its name as New, once we finished our code then we will delete the existing one and rename the New one to proper class name.